device/celestica/x86_64-cel_silverstone-r0/plugins/sfputil.py

```diff
@@ -344,11 +439,95 @@ def reset(self, port_num):
 
         return True
 
+    def _run_command(self, cmd):
+        status = True
+        result = ""
+        try:
+            p = subprocess.Popen(
+                cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+            raw_data, err = p.communicate()
+            if err == '':
+                result = raw_data.strip()
+        except Exception as e:
+            print "Error: unable to run command: %s" % str(e)
+            status = False
+        return status, result
+
+    def _bring_up_port_link(self, int_sfp, init_script):
```

Comment:
This should be renamed to something related to initializing the optical modules.